src/com/generic/main/Main.java

```diff
@@ -18,88 +19,241 @@
+					if (!warehouseTracker.isEmpty()) {
+						warehouseTracker.enableFreight(warehouseID1);
+						System.out.println("** Freight enabled for warehouse #" + warehouseID1);
+
+					} else {
+						System.out.println("** No warehouse avaliable, please import data first");
+					}
+				} catch (NumberFormatException | ArrayIndexOutOfBoundsException e) {
+					System.out.println("** Please input an integer");
+				}
+				break;
+				
+			case "disablef":
+				int warehouseID2;
+				try {
+					warehouseID2 = Integer.parseInt(arg[1]);
+					if (!warehouseTracker.isEmpty()) {
+						warehouseTracker.endFreight(warehouseID2);
+					} else {
+						System.out.println("** No warehouses avaliable, please import data first");
+					}
+				} catch (NumberFormatException | ArrayIndexOutOfBoundsException e) {
+					System.out.println("** Please input an integer");
+				}
+				break;
+
+			case "add":
+				try {
+					int mWarehouseID = Integer.parseInt(arg[1]);
+					if (!warehouseTracker.isEmpty()) {
+
+						if (!warehouseTracker.warehouseExists(mWarehouseID)) {
+							System.out.println("** Warehouse with ID #" + mWarehouseID + " does not exist");
+						} else {
+							boolean added = app.addShipmentOp(mWarehouseID , in);
+							if (!added) {
+								System.out.println("** Shipment could not be added because"
+										+ " freight has ended for this warehouse");
+							}
+						}
+					} else {
+						System.out.println("** No warehouses with avaliable, please import data first");
+					}
+				} catch (NumberFormatException | ArrayIndexOutOfBoundsException e) {
+					System.out.println("** Please input a warehouse_id");
+				}
+				break;
+			case "exit":
+				in.close();
+				break loop;
+			default:
+				System.out.println("** Invalid Command!");
+				continue;
 			}
 		}
+		
 		System.out.println("Goodbye!");
+		
+	}
+
+	/**
+	 * Adds shipments to the warehouse using the command line interface
+	 * 
+	 * @param warehouseID
+	 * @return
+	 */
+	public boolean addShipmentOp(int warehouseID , Scanner sc) {
+
+		
+		//Scanner sc = new Scanner(System.in);
+		boolean added = false;
+		String option = "";
+		
+
+		do {
+			
+			String shipmentID;
+			String shipmentString;
+			long receiptDate = 0;
+			double weight = 0;
+
+			
+			System.out.print("Enter shipmentID(i.e. iafr4, 15566) -> ");
+			shipmentID = sc.next();
+			
+			// TODO: Validate input for freight type
+			System.out.print("Enter shipment_method(i.e air, truck, ship, rail) -> ");
+			
+			ArrayList<String> shipmentMethodList = new ArrayList<String>(Arrays.asList(new String [] {FreightType.AIR.toString()
+																									, FreightType.RAIL.toString()
+																									, FreightType.TRUCK.toString()
+																									, FreightType.SHIP.toString()}));
+			shipmentString = sc.next().toUpperCase();
+			while(!shipmentMethodList.contains(shipmentString))
+			{
+				System.out.print("Please enter a valid shipment_method(i.e air, truck, ship, rail) -> ");
```

Comment:
Minor and have my approval. Just noting that we should stick to one convention of braces.